### ISSUE TITLE
Package guzzle/guzzle is abandoned, you should avoid using it. Use guzzlehttp/guzzle instead.

### ISSUE THREAD

**Issue by ghost** [FastFeed/FastFeed#32]
Are you still supporting this package?

**Comment by desarrolla2** [comment]
Hi. I'm sorry I'm not interested in this package right now. 

If one of you want to take control let me know.

**Comment by peter279k** [comment]
@desarrolla2, thank you for your reply.

If possible, I can maintain this and you can let me be the collaborator :+1:.

**Comment by desarrolla2** [comment]
I just send you the invitation.

Good luck

**Comment by KarelWintersky** [comment]
```
composer require fastfeed/fastfeed
Using version ^0.4.0 for fastfeed/fastfeed
./composer.json has been updated
Loading composer repositories with package information
Updating dependencies (including require-dev)
Package operations: 6 installs, 0 updates, 0 removals
  - Installing symfony/event-dispatcher (v2.8.49): Downloading (100%)         
  - Installing guzzle/guzzle (v3.9.3): Downloading (100%)         
  - Installing ezyang/htmlpurifier (v4.6.0): Downloading (100%)         
  - Installing fastfeed/url (v0.1.0): Downloading (100%)         
  - Installing desarrolla2/cache (v2.1.1): Downloading (100%)         
  - Installing fastfeed/fastfeed (v0.4.0): Downloading (100%)         
symfony/event-dispatcher suggests installing symfony/dependency-injection
symfony/event-dispatcher suggests installing symfony/http-kernel
guzzle/guzzle suggests installing guzzlehttp/guzzle (Guzzle 5 has moved to a new package name. The package you have installed, Guzzle 3, is deprecated.)
desarrolla2/cache suggests installing predis/predis (Predis support)
Package guzzle/guzzle is abandoned, you should avoid using it. Use guzzlehttp/guzzle instead.
```

bump

**Comment by peter279k** [comment]
Hi @KarelWintersky, thank you for your reply.

I will do this at my available time.

I also appreciate your concern and report.

**Comment by JorgRios** [comment]
hello, how can I do that I no longer ask for the dependency of guzzle and address to guzzlehttp, because when doing the update I'm still downloading the previous guzzle that is unsupported

**Comment by peter279k** [comment]
Hi @JorgRios, thank you for your reply.

Sorry for my delay, and please review the related PR.

If accepting the PR, I will merge this PR.

**Comment by peter279k** [comment]
Closed this issue via merged PR #33.
